code/index.html

```diff
@@ -1,22 +1,168 @@
+            <a class="header__nav__item" href="#" target="_blank">TECH</a>
+            <a class="header__nav__item" href="#" target="_blank">CULTURE</a>
+            <a class="header__nav__item" href="#" target="_blank">SOCIETY</a>
+            <a class="header__nav__item" href="#" target="_blank">SPORTS</a>
+            <a class="header__nav__item" href="#" target="_blank">BUSINESS</a>
+          </nav>
+    
+        </header>
+    
+        <!--FEATURE ARTICLE SECTION-->
+        <div class="Topcontent">
+    
+          <div class="Featured">
+            <h2 class="Featured__label">Featured Article</h2>
+            <div class="Featured__article">
+              <a class="Featured__image" href="#">
+                <img src="./assets/Crayfish.jpg" target="-blank" alt="Loading photo" height="400px">
+              </a>
+              <a class="Featured__link" href="#" target="-blank">Fullfil your summer with a crayfish party</a>
+              <p class="Featured__content">A highlight on the Swedish calendar, the ‘kräftskiva’ (crayfish party) – an annual sea food fest with lots of side dishes, drinks and joyous songs – is a summertime celebration dear to Swedes of all ages.</p>
+            </div>
+          </div>
+    
+          <!-- MOST READ SECTION-->
+          <div class="Mostread">
+            <h2 class="Mostread__label">Most Read</h2>
+            <ul class="Mostread__links">
+              <li><a href="#">Summer in Sweden -The top 10 tips</a></li>
+              <li><a href="#">Foreign students still in limbo after coronavirus stops Swedish embassy appointments</a></li>
+              <li><a href="#">A day or two in the archipelago</a></li>
+              <li><a href="#">Art in Subways of Stockholm</a></li>
+              <li><a href="#">Online Concert in Aug 2020</a></li>
+              <li><a href="#">17 great Swedish film acts</a></li>
+              <li><a href="#">Size of Sweden</a></li>
+              <li><a href="#">A Swedish fika in Stockholm</a></li>
+              <li><a href="#">25 landmarks of Swedish Architecture</a></li>
+            </ul>
+          </div>
+    
+          <!-- IMAGE SECTION-->
+          <div class="topleft">
+            <h2 class="topleft__label">Stockholm in me</h2>
+            <figure class="polaroid">
+              <img class="topleft__image" src="./assets/Stockholm image.jpg" alt="Stockholm moment" height="400">
+              <figcaption>
+              <p class="topleft__caption">Summer nights in Stockholm by @malun1973</p>
+              </figcaption>
+            </figure>
+          </div>
+    
+        </div>
+    
+        <!-- DISCOVER MORE SECTION-->
+        <h2 class= "discover__section__label">Discover more</h2>
+        <div class= "discover__section__layer1">
+    
+          <a class="discover__block" href="#">
+            <img class="discover__image" src="./assets/Exhibition.jpeg" alt="Exhibition in Stockholm right now" height="220">
```

Comment:
Great job using the alt on your img, it's to help describe your images and this is really nice!